Fortnox.NET/Services/OrderService.cs

```diff
@@ -61,5 +61,36 @@ public static async Task<Order> CreateOrderAsync(FortnoxApiRequest request, Orde
 
             return (await FortnoxAPIClient.CallAsync(apiRequest)).Data;
         }
+
+        /// <summary>
+        /// This action is used to set the field Sent as true from an external system without generating a PDF.
+        /// </summary>
+        /// <param name="request">FortnoxApiRequest object</param>
+        /// <param name="orderNumber">Fortnox Order Number</param>
+        /// <returns></returns>
+        public static async Task<Order> PrintAsync(FortnoxApiRequest request, int orderNumber)
```

Comment:
The name of this method might be confused with the actual print action for orders. Perhaps a name like `ExternalPrintAsync` is more fitting?